RELEASE_PROCEDURE.md

```diff
@@ -0,0 +1,29 @@
+# Release procedure
+
+When you release fixed version of SpotBugs, please follow these procedures.
+
+## Release to Maven Central
+
+Add necessary properties to `gradle.properties` and run `gradle clean build smoketest uploadArchives`.
+Make sure that you have proper `eclipsePlugin/local.properties` file to release Eclipse plugin at the same time.
+
+Check [SonaType official page](http://central.sonatype.org/pages/gradle.html) for detail.
+
+## Release to Eclipse Marketplace
+
+TBU
+
+## Release to Eclipse Update Site
+
+Send pull-request to spotbugs/spotbugs.github.io, to update contents.
+As files to upload, you can use the zip file which is uploaded to Maven Central when you release to there.
+
+See [this pull-request](https://github.com/spotbugs/spotbugs.github.io/pull/12) as example.
+
+## Release to Gradle Plugin Portal
+
+TBU
```

Comment:
Agreed, we should change the eclipse plugin id, this would also resolve #147.

Yes, I'll create a spotbugs account on Gradle plugin portal and email it to you.